### DISCUSSION TITLE
snowflake error when trying to execute fabric command

### DISCUSSION THREAD

**Discussion by mcelligottmike** [magma/magma#15237] [Q&A] [answered]
I have been following the quick start for 1.8. I have the AGW running in a VM fine and the orchestrator containers all up. When I try to run the fab command I am getting an error as below:

fab -f dev_tools.py register_vm

Fatal error: run() received nonzero return code 1 while executing!

Requested: cat /etc/snowflake
Executed: /bin/bash -l -c "cat /etc/snowflake"

============================= Standard output ============================
cat: /etc/snowflake: No such file or directory
====================================================================

Aborting.
Disconnecting from vagrant@127.0.0.1:2222... done.

Anyone have any suggestions about what I am missing here?
Thank you
Mike

**Comment by mcelligottmike** [comment, marked as answer]
I wiped magma .cache, reprovisioned magma VM (vagrant provision magma) and then 'make run' inside the VM. Now it is working fine so something I messed up myself

**Comment by panyogesh** [comment]
Closing the issue as its old and their is a solution possible.

**Reply by rajyay** [reply]
i got this error twice in this week inspite of going through the recommended solution
